src/AddTodoForm.js

```diff
@@ -9,9 +9,11 @@ function AddTodoForm({ onAddTodo }) {
   };
   const handleAddTodo = (event) => {
     event.preventDefault();
-    onAddTodo({ id: Date.now(), title: todoTitle });
-    console.log(todoTitle);
-    setTodoTitle('');
+    if (todoTitle.trim() !== '') {
```

Comment:
I like what I see here, we want  to make sure we don't save empty string in the todo list

Reply:
Let's start thinking about where in your code we could start adding these type of validations, 💡